docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md

```diff
@@ -0,0 +1,125 @@
+# Dasharo Compatibility: Logo customization functionality
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+
+## LCM001.001 Replace logo in existing image and flashing firmware
+
+**Test description**
+
+This test aims to verify whether it's possible to replace logo in existing
+firmware image and flash this firmware on the DUT.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Proceed with the
+    [Replace logo in an existing image](../../common-coreboot-docs/custom_logo.md#replace-logo-in-an-existing-image)
+    section.
+1. Reboot DUT.
+
+**Expected result**
+
+1. While booting custom logo appears on screen.
+
+## LCM002.001 Build image with custom logo and flashing firmware
+
+**Test description**
+
+This test aims to verify whether it's possible to build firmware image from the
```

Comment:
Is the purpose of this test to flash the device with the firmware? In my opinion, the primary goal is to check if the correct logo appears on the boot screen after flashing.

Reply:
Changed